tests-integration/src/utils/utils.hh

```diff
@@ -25,11 +25,44 @@ using namespace Mata::Nfa;
  * @param[out] aut Automaton instance to load into.
  * @param[out] alphabet Alphabet to use for symbols on transitions.
  * @param[in] skip_mintermization Whether to skip mirmization of the loaded automaton.
- * @param[in] aut_name Automaton name to print in logging outputs.
  * @return 0 if loading the automaton succeeded. Otherwise value != 0 if loading failed.
  */ 
-int load_automaton(const std::string& filename, Nfa& aut, Mata::OnTheFlyAlphabet& alphabet,
-                   const bool skip_mintermization = false, const std::string& aut_name = "unnamed");
+int load_automaton(
+        const std::string& filename,
+        Nfa& aut,
+        Mata::OnTheFlyAlphabet& alphabet,
+        const bool skip_mintermization = false
+);
+
+/**
+ * @brief Load automata from list of files at @p filename into list of automata @p aut,
+ * using @p alphabet for symbols on transitions.
+ *
+ * @param[in] filenames Paths to the file with automaton to load.
+ * @param[out] auts Vector of instances to load into.
+ * @param[out] alphabet Alphabet to use for symbols on transitions.
+ * @param[in] skip_mintermization Whether to skip mirmization of the loaded automaton.
```

Comment:
Mention here that if `skip_mintermization` is set to `false`, the automata are mintermized together and not separately.

Also, skip_mintermization is unnecessarily in a negative form. Shouldn't the parameter be `mintemize` with default to `true` (or `false`; whichever is supposed to be the default option). That is way more readable, in my opinion.

Reply:
The mintermization does not happen separately now, and I believe it should not happen separatly ever. If skip_mintermization is false, then no mintermization should happen at all. Probably it could be positive. I will change it.

Reply:
I meant that it might explain more clearly to the user, that he needs to keep in mind that the mintermization is not separate, as opposed to calling `load_automaton` twice.

Reply:
OK, I will improve the comment.